Account/FxAClient10.swift

```diff
@@ -362,4 +439,138 @@ public class FxAClient10 {
         }
         return deferred
     }
+
+    public func status(uid: String) -> Deferred<Maybe<FxAStatusResponse>> {
+        let deferred = Deferred<Maybe<FxAStatusResponse>>()
+
+        let baseURL = self.URL.URLByAppendingPathComponent("/account/status")
+        let queryParams = "?uid=" + uid
```

Comment:
Nit: Instead of using strings, there is an `NSURLQueryItem` object [1] you can use to add query params to a URL.

[1] https://developer.apple.com/library/mac/documentation/Foundation/Reference/NSURLQueryItem_Class/